### ISSUE TITLE
No response when card number is wrong with createTokenWithCard()

### ISSUE THREAD

**Issue by sonnt2510** [tipsi/tipsi-stripe#398]
hi guys, so basically i was installed stripe yesterday, and it worked quite well on both android and IOS, but i got an issue like this.
I used createTokenWithCard() and i tried test card number (4242 4242 4242 4242) with correct expiration date and cvc and it worked fine, it returned token for me. But when i changed to incorrect test card number or invalid expiration date, nothing happened. Even when i used catch, it didn't return error or anything. Here's my code, many thanks:

```
  stripe
        .createTokenWithCard(params)
        .then(response => {
          console.log("response", response);
        })
        .catch(error => {
          console.log("error", error);
        });
```
### Enviroment
"react-native": "^0.55.4"
"tipsi-stripe": "^5.6.0"

**Comment by yassinecc** [comment]
Hi @sonnt2510 , a similar issue (#372) has been reported, there are currently two work-in-progress PRs that aim to fix this problem, #392 and #377

**Comment by sonnt2510** [comment]
@yassinecc thanks for your replied. I'll look over

**Comment by theohdv** [comment]
It should not be closed as the issue still exists

**Comment by rupalpatel0008** [comment]
I am also having the same issue.

**Comment by giteekasawlani** [comment]
I am also facing same issue. Please anyone can tell me exact solution.

**Comment by sonnt2510** [comment]
Sorry for replied late, i also had this issue in ios only, but i followed this instruction [https://github.com/tipsi/tipsi-stripe/pull/392/commits/1f48b4f211ccd43e319e025effb2ab0db77a93cb](url) and i fixed it.

**Comment by giteekasawlani** [comment]
> Sorry for replied late, i had this issue in ios device only, and i fixed by followed this instruction (#392)
> You need to add `promiseRejector = reject;` in ios/TPSStripe/TPSStripeManager.m

Yes, I exactly did same, added code manually. and its working. Thanks for reply.

**Comment by pmatheusvinhas** [comment]
Hi guys. I'm facing that problem with Android on 2020. This correction was closed but pulled.
